scc/provider/resource_subaccount.go

```diff
@@ -227,6 +230,15 @@ func (r *SubaccountResource) Create(ctx context.Context, req resource.CreateRequ
 		return
 	}
 
+	// Trigger trust configuration sync for the subaccount without persisting to Terraform state
```

Comment:
This can be made into a separate helper function, considering it's the same call in Create, Read and Update